ds003/models/model-001_smdl.json

```diff
@@ -6,7 +6,7 @@
   },
   "Steps": [
     {
-      "Level": "run",
+      "Level": "Run",
```

Comment:
@tyarkoni Was the desire to turn all strings into CamelCase, or just keys? As levels match with BIDS-Apps analysis_level arguments, I thought it made sense to leave these lowercase.